spec/ios_get_app_version_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::IosGetAppVersionAction do
+  describe 'getting the public app version from the provided .xcconfig file' do
+    it 'parses the xcconfig file format correctly and gets the public version' do
+      xcconfig_mock_content = <<~CONTENT
+        // a comment
+        VERSION_SHORT=6
+        VERSION_LONG=6.30.0
+      CONTENT
+
+      allow(File).to receive(:exist?).and_return(true)
```

Comment:
I started commenting about this `allow` being a bit too loose and duplicated, but I see that https://github.com/iangmaia/release-toolkit/pull/1 extracted it in the `expect_version` helper (great idea).  I'll comment there.

Reply:
https://github.com/iangmaia/release-toolkit/pull/1/files#r1094033726